wikimaginot/import_maginot.py

```diff
@@ -0,0 +1,196 @@
+import argparse
+import csv
+import glob
+import os
+import requests
+from pathlib import Path
+
+
+##### dossiers ######
+# on va stocker les photos dans files/IGN/ et files/OSM/ :
+s_stockage = "files"
+s_dossier_ign = "IGN"
+s_dossier_osm = "OSM"
+##### instances #####
+s_instance = "http://localhost:5000/"
+s_token = ""
+s_instance_ign = "https://panoramax.ign.fr"
+s_token_ign = "--token \"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnZW92aXNpbyIsInN1YiI6ImUxOWRiMTM3LTI3ZGMtNDJlNy1iZWZlLWRhMTg5YzQwNzFjNSJ9.0Wcv1ObVysSmBrsJDP_JxgEOx0NAdtC1okKyhcPC3Bk\" "
```

Comment:
This may not be kept in the example, as the token allows anyone to upload... :grin:

Reply:
oh no, we need to invalidate this token...